packages/core/src/App/Containers/Modals/app-modals.jsx

```diff
@@ -47,6 +51,12 @@ const AppModals = ({
         case 'signup':
             ComponentToLoad = <AccountSignupModal />;
             break;
+        case 'request_email':
+            ComponentToLoad = <ResetEmailModal />;
+            break;
+        case 'system_email_change':
+            ComponentToLoad = <UpdateEmailModal />;
```

Comment:
after setting the email, we send a email to the new email and on clicking that the user is redirected to `success` or `fail`modal and sending the `update` email request in the background.